lib/dor/was_seed/desc_metadata_generator.rb

```diff
@@ -5,8 +5,8 @@ module WasSeed
     class DescMetadataGenerator < MetadataGenerator
       attr_reader :desc_metadata_name, :seed_uri, :collection_id
 
-      def initialize(workspace_path, druid_id, seed_uri, collection_id)
-        super(workspace_path, druid_id)
+      def initialize(workspace_path, druid_id, seed_uri, collection_id, logger: nil)
```

Comment:
was robots need their own logger?

Reply:
The old lyber-core used a global logger. The new lyber-core uses Sidekiq's logger, which must be passed around.